tests/Benchmarks/Buffers/Reader.cs

```diff
@@ -0,0 +1,65 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using System;
+using System.Buffers;
+using System.Buffers.Reader;
+using System.Buffers.Text;
+using System.Text;
+
+public class Reader
```

Comment:
@adamsitnik, I think I don't like the convention of calling the test types with a single word, e.g. "Reader". The reason is that the type names are used as the "menu" items in the BenchmarkSwitcher's menu. I suspect we will have several readers and it will be impossible to tell them apart in the menu.